src/nectarchain/dqm/bokeh_app/main.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+
+# bokeh imports
+from bokeh.layouts import layout, row
+from bokeh.models import Select  # , NumericInput
+from bokeh.plotting import curdoc
+
+# ctapipe imports
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+
+from ctapipe_io_nectarcam import constants
+
+from nectarchain.dqm.db_utils import DQMDB
+from app_hooks import get_rundata, make_camera_displays
+
+NOTINDISPLAY = ['Results_TriggerStatistics',
+                'Results_MeanWaveForms_HighGain',
+                'Results_MeanWaveForms_LowGain',
+                'Results_CameraMonitoring']
+
+geom = CameraGeometry.from_name("NectarCam-003")
```

Comment:
Should the camera geometry to use be a configurable parameter?

Reply:
Or I guess, we could just use a more generic `NectarCam` parameter, which should work.